homework/style.css

```diff
@@ -1,3 +1,60 @@
+* {
```

Comment:
On a mobile phone you are wasting a lot of screen real estate. Better trim down on the left and right margins.

![image](https://user-images.githubusercontent.com/2788771/65241326-8e385a80-dae3-11e9-837f-2f38d51573d2.png)

Also, your error message box is a bit cramped:

![image](https://user-images.githubusercontent.com/2788771/65241371-a4deb180-dae3-11e9-84d2-38a056dda283.png)

Reply:
I made the necessary corrections as you suggested. Thanks a lot.